composer.json

```diff
@@ -9,7 +9,7 @@
         }
     ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": ">=5.6",
```

Comment:
Inspired by other libraries like [symfony](https://github.com/symfony/symfony/blob/0ab5eeddf97d0a277c5ad956a51365aba93b27b9/composer.json#L19) to use >=  instead of ^ for php version.